editor/src/core/model/element-metadata-utils.ts

```diff
@@ -2000,6 +2015,59 @@ export const MetadataUtils = {
       return false
     }
   },
+  getEmphasisOfComponent(
+    path: ElementPath,
+    metadata: ElementInstanceMetadataMap,
+    propertyControlsInfo: PropertyControlsInfo,
+    projectContents: ProjectContentTreeRoot,
+  ): Emphasis {
+    // Look up the emphasis of the component from the property controls.
+    const componentDescriptor = getComponentDescriptorForTarget(
+      path,
+      propertyControlsInfo,
+      projectContents,
+    )
+    if (componentDescriptor != null) {
+      return componentDescriptor.emphasis
+    }
+
+    const element = MetadataUtils.findElementByElementPath(metadata, path)
+    // Element with flex or grid get high emphasis.
+    if (
+      MetadataUtils.isFlexLayoutedContainer(element) ||
+      MetadataUtils.isGridLayoutedContainer(element)
+    ) {
+      return 'emphasized'
+    }
+
+    // Divs without styling that contain one or zero elements get low emphasis.
+    if (element != null && MetadataUtils.isDiv(element)) {
+      const children = MetadataUtils.getChildrenUnordered(metadata, path)
+      if (children.length <= 1) {
+        const attributes = toFirst(
+          fromField<ElementInstanceMetadata, 'element'>('element')
+            .compose(eitherRight())
+            .compose(fromTypeGuard(isJSXElement))
+            .compose(fromField('props')),
+          element,
+        )
+        if (isRight(attributes)) {
+          const styleAttributes = getJSXAttribute(attributes.value, 'style')
+          if (styleAttributes != null) {
```

Comment:
```suggestion
          if (styleAttributes == null) {
```
I'm pretty sure this is a typo?

Reply:
I think you're right there.